reposerver/repository/repository_test.go

```diff
@@ -132,7 +132,7 @@ func TestGenerateYamlManifestInDir(t *testing.T) {
 	q := apiclient.ManifestRequest{Repo: &argoappv1.Repository{}, ApplicationSource: &src}
 
 	// update this value if we add/remove manifests
-	const countOfManifests = 47
```

Comment:
I think we should really get rid of this test one day or use a "stable" source repository.

Reply:
agree, this is pretty annoying test